db/re/mob_db.yml

```diff
@@ -82968,120 +82968,2610 @@ Body:
 #    AegisName: WANDERING_SOUL
 #  - Id: 3383
 #    AegisName: ANGRY_PENGUIN
-#  - Id: 3384
-#    AegisName: MIN_PORING
-#  - Id: 3385
-#    AegisName: MIN_LUNATIC
-#  - Id: 3386
-#    AegisName: MIN_FABRE
-#  - Id: 3387
-#    AegisName: MIN_PICKY
-#  - Id: 3388
-#    AegisName: MIN_CONDOR
-#  - Id: 3389
-#    AegisName: MIN_WILOW
-#  - Id: 3390
-#    AegisName: MIN_SPORE
-#  - Id: 3391
-#    AegisName: MIN_POPORING
-#  - Id: 3392
-#    AegisName: MIN_SMOKIE
-#  - Id: 3393
-#    AegisName: MIN_DOKEBI
-#  - Id: 3394
-#    AegisName: MIN_BIGFOOT
-#  - Id: 3395
-#    AegisName: MIN_WORM_TAIL
-#  - Id: 3396
-#    AegisName: MIN_WOLF
-#  - Id: 3397
-#    AegisName: MIN_SNAKE
-#  - Id: 3398
-#    AegisName: MIN_ANACONDAQ
-#  - Id: 3399
-#    AegisName: MIN_MARIN
-#  - Id: 3400
-#    AegisName: MIN_MUKA
-#  - Id: 3401
-#    AegisName: MIN_PECOPECO
-#  - Id: 3402
-#    AegisName: MIN_DENIRO
-#  - Id: 3403
-#    AegisName: MIN_PIERE
-#  - Id: 3404
-#    AegisName: MIN_ANDRE
-#  - Id: 3405
-#    AegisName: MIN_GOLEM
-#  - Id: 3406
-#    AegisName: MIN_SCORPION
-#  - Id: 3407
-#    AegisName: MIN_CHONCHON
-#  - Id: 3408
-#    AegisName: MIN_METALLER
-#  - Id: 3409
-#    AegisName: MIN_SAND_MAN
-#  - Id: 3410
-#    AegisName: MIN_RAGGLER
-#  - Id: 3411
-#    AegisName: MIN_DRAGON_TAIL
-#  - Id: 3412
-#    AegisName: MIN_GREEN_IGUANA
-#  - Id: 3413
-#    AegisName: MIN_SEE_OTTER
-#  - Id: 3414
-#    AegisName: MIN_GALAPAGO
-#  - Id: 3415
-#    AegisName: MIN_FUR_SEAL
-#  - Id: 3416
-#    AegisName: MIN_ALLIGATOR
-#  - Id: 3417
-#    AegisName: MIN_MEGALODON
-#  - Id: 3418
-#    AegisName: MIN_TRI_JOINT
-#  - Id: 3419
-#    AegisName: MIN_MEGALITH
-#  - Id: 3420
-#    AegisName: MIN_DRYAD
-#  - Id: 3421
-#    AegisName: MIN_TOAD
-#  - Id: 3422
-#    AegisName: MIN_VAGABOND_WOLF
-#  - Id: 3423
-#    AegisName: MIN_VOCAL
-#  - Id: 3424
-#    AegisName: MIN_ECLIPSE
-#  - Id: 3425
-#    AegisName: MIN_CHIMERA
-#  - Id: 3426
-#    AegisName: MIN_EDDGA
-#  - Id: 3427
-#    AegisName: MIN_OSIRIS
-#  - Id: 3428
-#    AegisName: MIN_PHREEONI
-#  - Id: 3429
-#    AegisName: MIN_ORK_HERO
-#  - Id: 3430
-#    AegisName: MIN_TAO_GUNKA
-#  - Id: 3431
-#    AegisName: MIN_G_RODA_FROG
-#  - Id: 3432
-#    AegisName: MIN_G_WOLF
-#  - Id: 3433
-#    AegisName: MIN_G_ROCKER
-#  - Id: 3434
-#    AegisName: MIN_G_LUNATIC
-#  - Id: 3435
-#    AegisName: MIN_G_GARGOYLE
-#  - Id: 3436
-#    AegisName: MIN_G_BIGFOOT
-#  - Id: 3437
-#    AegisName: MIN_G_ANCIENT_MUMMY
-#  - Id: 3438
-#    AegisName: MIN_G_SAND_MAN
-#  - Id: 3439
-#    AegisName: MIN_G_HIGH_ORC
-#  - Id: 3440
-#    AegisName: MIN_G_MEGALITH
+  - Id: 3384
+    AegisName: MIN_PORING
+    Name: Poring
+    Level: 100
+    Hp: 25000
+    BaseExp: 1200
+    JobExp: 1200
+    Attack: 843
+    Attack2: 1160
```

Comment:
![Capture d’écran (612)](https://user-images.githubusercontent.com/5231893/149001600-d1e492da-43bc-4a1a-8a58-44b018d1ecc1.png)
The values displayed on DP are the final values, we need the "raw" values for Attack and Attack2.
You can find the raw values on DP API, here it should be `Attack: 794` and `Attack2: 67` for MIN_PORING

![Capture d’écran (611)](https://user-images.githubusercontent.com/5231893/149001943-24a1cf91-35aa-4670-96d7-85c2e009093c.png)
The Attack/Attack2 need to be corrected for all monsters

Reply:
How can I check the DP API?

Reply:
> How can I check the DP API?

https://www.divine-pride.net/api
here